scripts/trace_scenario.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
Let's have a single way to do the TV and improve it if needed, but avoid two parallel paths that may or may not check exactly the same things.